### PR TITLE
feat: article page refactor to functional componet

### PR DESCRIPTION
**Pull request by liruchen32**
# Issue
Refactor the article page into a functional component to see if it improves prevention of reading time errors.

# Notice

# Dependency

### CODE REVIEW

File: src/containers/Article-new.js [hunk 1]
```diff
@@ -0,0 +1,617 @@
...
+      document.removeEventListener('scroll', handleScroll)
+      document.removeEventListener('visibilitychange', handleVisibilityChange)
+      document.removeEventListener('mousemove', handleUserActivity)
+      document.removeEventListener('click', handleUserActivity)
+      window.removeEventListener('pagehide', handlePagehide)
+    }
+  }, [])
+
+  useEffect(() => {
+    if (!isFetchingPost) {
```
Comment (Aylie-Chou) [inline src/containers/Article-new.js:298]: The default value of `isFetchingPost` is false, which means it's value might change like : false -> true -> false
Do we need to check the previous props value in order to match the false after true ?

File: src/containers/Article-new.js [hunk 2]
```diff
@@ -0,0 +1,617 @@
...
+  reading_height: 0.75,
+  reading_time: 10000, // 10 second
+}
+const articleReadTimeConditionConfig = {
+  inactive_time: 900000, // 900 second
+  min_active_time: 5000, // 5 second
+}
+
+const Article = ({
+  slugToFetch,
```
Comment (Aylie-Chou) [inline src/containers/Article-new.js:54]: Nice to have:
Could we arrange the props to show that which is passed by redux state (mapStateByProps) & by parent

**Review by Aylie-Chou — CHANGES_REQUESTED** [review]
Nice work to refactor article page !
This page is important for our users

Please have a look at comments ~~

File: src/containers/Article-new.js (hunk 1 above)
Comment (liruchen32) [reply src/containers/Article-new.js]: fixed

File: src/containers/Article-new.js (hunk 2 above)
Comment (liruchen32) [reply src/containers/Article-new.js]: yes sir

**Review by Aylie-Chou — APPROVED** [review]
LGTM

**Pull request merged**